content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -79,51 +79,51 @@ The pod network plugin you use (see below) may also require certain ports to be
 documentation for the plugins about what port(s) those need.
 
-## Installing runtime {#installing-runtime}
+## Installing a container runtime {#installing-runtime}
 
 To run containers in Pods, Kubernetes uses a
 {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
 
-{{< tabs name="container_runtime" >}}
-{{% tab name="Linux nodes" %}}
-
 By default, Kubernetes uses the
 {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
 to interface with your chosen container runtime.
 
 If you don't specify a runtime, kubeadm automatically tries to detect an installed
-container runtime by scanning through a list of well known Unix domain sockets.
-The following table lists container runtimes and their associated socket paths:
-
-{{< table caption = "Container runtimes and their socket paths" >}}
-| Runtime    | Path to Unix domain socket        |
-|------------|-----------------------------------|
-| Docker     | `/var/run/dockershim.sock`        |
-| containerd | `/run/containerd/containerd.sock` |
-| CRI-O      | `/var/run/crio/crio.sock`         |
-{{< /table >}}
-
-<br />
-If both Docker and containerd are detected, Docker takes precedence. This is
-needed because Docker 18.09 ships with containerd and both are detectable even if you only
-installed Docker.
-If any other two or more runtimes are detected, kubeadm exits with an error.
+container runtime by scanning through a list of known endpoints.
 
-The kubelet integrates with Docker through the built-in `dockershim` CRI implementation.
+If multiple or no container runtimes are detected kubeadm will throw an error
+and will request that you specify which one you want to use.
 
 See [container runtimes](/docs/setup/production-environment/container-runtimes/)
 for more information.
+
+This tables below include the known endpoints for supported operating systems:
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux" %}}
+
+{{< table >}}
+| Runtime      | Path to Unix domain socket                   |
+|--------------|----------------------------------------------|
+| containerd   | `unix:///var/run/containerd/containerd.sock` |
+| CRI-O        | `unix:///var/run/crio/crio.sock`             |
+| Docker       | `unix:///var/run/cri-dockerd.sock`           |
+{{< /table >}}
+
 {{% /tab %}}
-{{% tab name="other operating systems" %}}
-By default, kubeadm uses {{< glossary_tooltip term_id="docker" >}} as the container runtime.
-The kubelet integrates with Docker through the built-in `dockershim` CRI implementation.
 
-See [container runtimes](/docs/setup/production-environment/container-runtimes/)
-for more information.
+{{% tab name="Windows" %}}
+
+{{< table >}}
+| Runtime      | Path to Windows named pipe                   |
+|--------------|----------------------------------------------|
+| containerd   | `npipe:////./pipe/containerd-containerd`     |
+| Docker       | `npipe:////./pipe/cri-dockerd`               |
```

Comment:
AFAIK nobody has tested out cri-dockerd on Windows.
Should be adding an entry for cri-dockerd here (in the 'production-environment') section knowing this?

Reply:
That is true. Cri-dockerd is so new that likely nobody has tested it on Windows yet.

We have a few options:
1. Delete the Docker information until further notice.
2. Merge this as is, and start testing it at some point.
3. Add a warning that cri-dockerd is a new project.

My vote goes for 3.

Reply:
It was launched in April 2021, but I guess that is still rather new (in Kubernetes years).

Same could be said for Linux, but removing the documentation only makes it less tested ?

Reply:
+1 Add a warning that cri-dockerd is a new project. (A new project which uses old codes. 😂)

Reply:
+1 for option #3 
We can remove at a future point in time.